main/app-menu.js

```diff
@@ -14,6 +15,12 @@ function setupAppMenu (/** @type {import('./typings').Context} */ ctx) {
     label: 'Save Module Logs As…',
     click: () => { ctx.saveModuleLogsAs() }
   }))
+  menu.items[1].submenu?.insert(1, new MenuItem({
+    label: 'Copy Seed Phrase To Clipboard',
```

Comment:
Will Station users understand what "Seed Phrase" refers to?

```suggestion
    label: 'Copy Wallet Seed Phrase To Clipboard',
```